Bottom: 0579e5b4aae4e538bd9c696446700d9bc07d8840
Top:    a3d5c72f9f0aa5302384206c0453882dd209a70d
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 14:09:53 +0800

Refresh of autofs-5.0.7-fix-libtirpc-build-option.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 8fa31fc..25aa5fc 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -33,6 +33,7 @@
 - depricate nosymlink pseudo option.
 - add symlink pseudo option.
 - fix requires in spec file.
+- fix libtirpc build option to require libtirpc-devel if needed.
 
 25/07/2012 autofs-5.0.7
 =======================
